docs/getting-started.md

```diff
@@ -75,18 +75,25 @@ You can use Ubuntu 20.04 LTS and Ubuntu 22.04 LTS inside Windows through Windows
 4. Click the `Fork` button in the top-right to fork the repository ([click here to learn about Forks](https://help.github.com/en/articles/fork-a-repo))
    1. Click on your user when prompted
    2. You should be automatically redirected to your new fork
-5. Clone your fork of the repository. As GitHub is forcing users to stop using usernames and passwords, we will be using the SSH link.  Returning members who are migrating to using SSH after cloning from a previous method can use the following instructions to set up a new local repository using SSH.
-   1. To connect to GitHub using SSH, if not setup prior, you will need to add an SSH key to your GitHub account. Instructions can be found [here](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent).  For each computer you contribute to GitHub with, you will need an additional SSH Key pair linked to your account.
-   2.  After you have successfully set up a SSH key for your device and added it to GitHub, you can clone the repository using the following command (you can put it wherever you like):
-        1.  Eg. `git clone git@github.com:<your_username>/Software.git`
-        2.  You can find this link under the green `Clone or Download` button on the main page of the Software repository, under the SSH tab.  (This should now be available after adding your SSH key to GitHub successfully.)
+5. Clone your fork of the repository. We recommend cloning using HTTPS since it is the easiest to set up. You should be able to clone your fork using the following command (you can put it wherever you like):
+   ```
+   git clone https://github.com/<your_username>/Software.git
+   ```
+   You can find your fork's remote URL under the green `Code` button on the main page of your fork on GitHub, under the HTTPS tab.
```

Comment:
we shouldn't show HTTPS cloning. You won't be able to push/pull because github doesn't do password authentication anymore.

Reply:
well... I use https and it's easy peasy. But I forgot that I have a credential helper (GitHub CLI). Once you run `gh auth login` GitHub opens up in your browser and prompts you to sign in with 2fa. Then your credentials are cached and push/pull via https never prompts you for credentials

Reply:
Ok, we can keep ssh as the default. I was just thinking of suggesting https since setting up ssh is not exactly straightforward either